lib/minitag/minitest_tag.rb

```diff
@@ -36,6 +36,21 @@ def tag(*tags)
       Minitag.register_for_extension(self)
     end
 
+    def run_one_method(klass, method_name, reporter) # rubocop:disable Metrics/AbcSize
+      return super if !Minitag.skip_filtered? || Minitag.context.no_filters? ||
+                      Minitag.context.match?(namespace: to_s, name: method_name)
+
+      # Skip this test
+      test = klass.new(method_name)
+      test.time = 0
```

Comment:
This is the only line I'm unsure of. Do we still need this even though we are not calling the `run` method in the test class?

Reply:
Yes, this is used in the verbose report. You'll see things like this:

```
MyTest#test_foo = 0.00 s = S
```

If left unspecified, minitest throws an error.